ts/components/ui/RefreshIndicator.tsx

```diff
@@ -0,0 +1,27 @@
+import { View } from "native-base";
+import * as React from "react";
+import { ActivityIndicator, Platform, StyleSheet } from "react-native";
+
+import variables from "../../theme/variables";
+
+const styles = StyleSheet.create({
+  androidIndicatorInner: {
+    justifyContent: "center",
+    backgroundColor: variables.colorWhite,
+    width: 42,
+    height: 42,
+    elevation: 4,
+    borderRadius: 42
+  }
+});
+
+const isAndroid = Platform.OS === "android";
+
+export const RefreshIndicator: React.SFC<{}> = () => (
+  <View style={isAndroid && styles.androidIndicatorInner}>
```

Comment:
isn't this redundant with the `<View style={styles.refreshIndicatorContainer}>` below?

also, since the View is only needed for Android, you could avoid the extra view on iOS:

```ts
const activityIndicator = <ActivityIndicator ... />;
return isAndroid ? <View style={isAndroid && styles.androidIndicatorInner}>{activityIndicator}</View> : activityIndicator;
```